ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnerGroups.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div id="mhr-home-owner-groups">
+    <label class="generic-label">
+      Multiple Groups of Owners (Tenants in Common)
+    </label>
+    <ul class="my-2">
+      <li>
+        Select a group if you have
+        <b>multiple groups of owners</b> (tenants in common).
+      </li>
+      <li>
+        Leave this empty if you have <b>only one owner</b>, or
+        <b>one group of owners</b> (sole ownership or joint tenancy).
+      </li>
+    </ul>
+    <v-select
+      id="home-owner-groups"
+      label="Select a Group"
+      v-model="ownerGroupId"
+      :items="groups"
+      :rules="groupRules"
+      class="owner-groups-select my-8"
+      filled
+      @change="setOwnerGroupId($event)"
+      :clearable="groups.length === 1"
+    ></v-select>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  reactive,
+  toRefs
+} from '@vue/composition-api'
+import { useHomeOwners } from '@/composables/mhrRegistration'
+import { useInputRules } from '@/composables'
+
+export default defineComponent({
+  name: 'HomeOwnerGroups',
+  props: {
+    groupId: { type: String },
+    isAddingHomeOwner: { type: Boolean } // make additional Group available in dropdown when adding a new home owner
+  },
+  setup (props, { emit }) {
+    const { required } = useInputRules()
+    const { getGroupDropdownItems, showGroups } = useHomeOwners()
+
+    const localState = reactive({
+      ownerGroupId: props.groupId,
+      groups: computed(() => getGroupDropdownItems(props.isAddingHomeOwner)),
+      groupRules: computed(() =>
+        showGroups.value ? required('Select a group for this owner') : []
+      )
+    })
+
+    const setOwnerGroupId = groupId => {
+      emit('setOwnerGroupId', groupId)
+    }
```

Comment:
While i'm not opposed to 'less is more', for consistency sake might be a good idea to add the `()` and an argument type, `string` in this case.

Reply:
Added.